example/android/app/build.gradle

```diff
@@ -137,14 +136,17 @@ android {
             }
         }
     }
+
+    compileOptions {
+        sourceCompatibility 1.8
+        targetCompatibility 1.8
+    }
 }
 
 dependencies {
-    compile project(':lottie-react-native')
-    compile 'com.android.support:appcompat-v7:26.1.0'
-    compile 'com.android.support:support-annotations:26.1.0'
-    compile "com.facebook.react:react-native:+"  // From node_modules
-
+    implementation project(':lottie-react-native')
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
Great idea with the update! But I have some reservations about updating RN libraries to `AndroidX`. 
I personally would love if everyone would switch to androidx all at once so we could avoid fragmentation as much as possible., but it's just not feasible. Right now 99% of RN libraries do not support android jetpack so adding dependency with androidx import is just gonna break about every app.
Can't help but notice that `lottie-react-native` uses `Support-v4` only for `ViewCompat.isAttachedToWindow()` in line https://github.com/react-native-community/lottie-react-native/blob/master/src/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManager.java#L118 and https://github.com/react-native-community/lottie-react-native/blob/master/src/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManager.java#L144
I propose to replace this call with https://developer.android.com/reference/android/view/View.html#isAttachedToWindow(), so it's gonna be `view.isAttachedToWindow`, and removing `support-v4` dependency altogether. Only downside to this is loosing compatibility with android apis 16-19 (so about 3.2% of Android devices https://developer.android.com/about/dashboards) but you gain compatibility with AndroidX _and_ Android Support Library projects, and thats huge!

Reply:
Anychance of this, getting merged without the change in SDK? It seriously is causing issues.

Reply:
@rozPierog I'm not sure if I get your suggestion. Could you send a PR with it so I can give it a try? Thanks!

Reply:
@influx6 what kind of issues current `lottie-react-native` is causing at the moment? As I mentioned, I'm a bit hesitant to go ahead with this PR as per @rozpierog comment

Reply:
I also won't advise it due to androidx support libraries. We resolved our issues, as we realized we were exported in bodymovin with lottie3 format

Reply:
@influx6 Your comment was incredibly helpful. Almost all the animations on the LottieFiles Community site were working but even the simplest scaling animations of mine weren't working. Turns out it's the setting you mentioned under Advanced Settings. Same solution as to @airbnb/lottie-web#1561 incase anyone else stumbles upon this. It looks like the documentation has been updated to say the current version requires AndroidX? This was confusing.